packages/api/src/middleware/sessionMiddleware.ts

```diff
@@ -3,15 +3,15 @@ import { Handler } from 'express';
 /**
  * A middleware handler to check to see if the request included a valid session.
  *
- * Session validity is determined by the presence of an email within signed cookie included with the request.
+ * Session validity is determined by the presence of an email and id within signed cookie included with the request.
  * If the session is valid, the next function is invoked.
  *
  * @param req Express Request
  * @param res Express Response
  * @param next Express Next function
  */
 export const sessionMiddleware: Handler = (req, res, next) => {
-  if (!req.session?.email) {
+  if (!req.session?.email || !req.session?.id) {
```

Comment:
This breaks the existing functionality; now ID is required when creating a user which will always fail